proto/plugin.proto

```diff
@@ -862,7 +891,7 @@ service ReleaseManager {
   // component.Destroyer optional implementation
   rpc IsDestroyer(google.protobuf.Empty) returns (ImplementsResp);
   rpc DestroySpec(google.protobuf.Empty) returns (FuncSpec);
-  rpc Destroy(FuncSpec.Args) returns (google.protobuf.Empty);
+  rpc Destroy(FuncSpec.Args) returns (Destroy.Resp);
```

Comment:
After looking over the protos, I think it's safe to leave this as-is since we could also just add a field in `Resp`! Aside from that, it seems like other things which use `Resp` like the `Deploy` rpc don't change very often. 😄